packages/wonder-blocks-data/src/util/response-cache.js

```diff
@@ -101,6 +83,8 @@ export class ResponseCache {
 
     /**
      * Cache data for a specific response.
+     *
+     * This is a noop when client-side.
```

Comment:
Thanks for adding in comments like this to indicate differences in behaviour between client and server.